index.js

```diff
@@ -110,10 +110,11 @@ module.exports = function (filename, options = {}) {
   })
 
   const loggers = {
+    debug: makeStrictLogger(`${prefix}:DEBUG:${relativeFilePath}`, options),
     log: makeStrictLogger(`${prefix}:LOG:${relativeFilePath}`, options),
+    warn: makeStrictLogger(`${prefix}:WARN:${relativeFilePath}`, options),
     error: makeStrictLogger(`${prefix}:ERROR:${relativeFilePath}`, errorOptions),
-    debug: makeStrictLogger(`${prefix}:DEBUG:${relativeFilePath}`, options),
-    warn: makeStrictLogger(`${prefix}:WARN:${relativeFilePath}`, options)
+    emergency: makeStrictLogger(`${prefix}:EMERGENCY:${relativeFilePath}`, errorOptions),
```

Comment:
do we want this to be `emergency` or `fatal`? (as `fatal` is the standard for the highest log level)

Reply:
@konradstrachan suggested `EMERGENCY` level for GCP to handle as urgent, not sure tho if thats automatic?

Reply:
I have tied the emergency level to alerting so this is the right level to use